shell/src/test/java/tachyon/shell/TFsShellTest.java

```diff
@@ -337,94 +345,89 @@ public void locationNotExistTest() throws IOException {
     Assert.assertEquals(-1, ret);
   }
 
-  @Test
-  public void locationTest() throws IOException {
-    TachyonFSTestUtils.createByteFile(mTfs, "/testFile", WriteType.MUST_CACHE, 10);
-    mFsShell.run(new String[] {"location", "/testFile"});
-    String locationResult = getLocationResultStr("/testFile");
-    Assert.assertEquals(locationResult, mOutput.toString());
-  }
-
-  private String getLocationResultStr(String path) throws IOException {
-    TachyonURI tUri = new TachyonURI(path);
-    TachyonFile tFile = mTfs.getFile(tUri);
-    Assert.assertNotNull(tFile);
-    List<String> locationsList = tFile.getLocationHosts();
-    String[] commandParameters = new String[3 + locationsList.size()];
-    commandParameters[0] = "location";
-    commandParameters[1] = path;
-    commandParameters[2] = String.valueOf(mTfs.getFileId(tUri));
-    Iterator<String> iter = locationsList.iterator();
-    int i = 3;
-    while (iter.hasNext()) {
-      commandParameters[i ++] = iter.next();
-    }
-    return getCommandOutput(commandParameters);
-  }
-
+  /*
+   * @Test public void locationTest() throws IOException { TachyonFile file =
```

Comment:
Should there be a TODO describing what to do with this test ... also the formatting seems off